test/RegisterDomains.ts

```diff
@@ -0,0 +1,94 @@
+const { expect } = require("chai");
+const {
+    loadFixture,
+} = require("@nomicfoundation/hardhat-toolbox/network-helpers");
+const { ethers } = require("hardhat");
+
+const undefinedAddress = "0x0000000000000000000000000000000000000000";
```

Comment:
There should be a `ZeroAddress` constant in `ethers.js`

Reply:
fixed: imported from `@ethersproject/constants`